### ISSUE TITLE
Index of Ligatures

### ISSUE THREAD

**Issue by emylonas** [MedCyprus/MedCyprusEFES#1]
1. pulling from wrong attribute value (clarify)
2. Future: ligatures across words
3. Future: accented letters (need explanation, example)

**Comment by emylonas** [comment]
043.17 - Μήτηρ shouldn't be the ligature. It's ΜΗΡ. check

**Comment by TheocharisPetrou** [comment]
I agree.  I checked the mark up of all the records and I think I marked up them correct. Perhaps the problem has to do with the fact that ΜΗΡ is ligature and at the same time it is also an abbreviation: Μή(τη)ρ / Μῆ(τε)ρ.

**Comment by gabrielbodard** [comment]
In other words, each entry in the ligature index is triggered by one of the following occurances:
* `<hi rend="ligature">ABC</hi>`
* `<hi rend="ligature" xml:id="h001">MH</hi>TH<hi rend="ligature" xml:id="h002">P</hi><join targets="#h001 #h002"/>`

**Comment by emylonas** [comment]
non-diplomatic elements like `<ex>` and `<corr>` won't appear, but `<supplied reason="lost">` should show up in the index.

**Comment by gabrielbodard** [comment]
Correct. However…

For the particular of the encoding of ligatures, our practice needs to be that if two parts of a ligature are divided **in the edition**, by any markup, spaces, restored/expanded/corrected characters, or anything else, it should be tagged as two `<hi>` elements, with a `<join>`. (As in my second example above.)

**Comment by gabrielbodard** [comment]
See e.g. [044.001](https://medcyprus.ucy.ac.cy/en/inscriptions/MedCY.044.001.html): markup is `<hi rend="ligature" xml:id="lig1"><choice><orig>Η</orig><reg>Ἡ</reg></choice></hi> <addName type="epithet" key="Pantanassa"><hi rend="ligature" xml:id="lig2">Π</hi><join target="#lig1 #lig2"/>` (CORRECT), but H and Π appear as two separate ligatures in the index (and in the EpiDoc display, NB).

However in [060.001](https://medcyprus.ucy.ac.cy/en/inscriptions/MedCY.060.001.html) the markup is incorrect, so not a good diagnostic of broken index. To check with Theo and Dora.

**Comment by gabrielbodard** [comment]
I have fixed the incorrect markup in [043,017](https://medcyprus.ucy.ac.cy/en/inscriptions/MedCY.043.017.html) to read: `<expan><abbr rend="supraline"><hi rend="ligature">Μ<choice><orig>η</orig><reg>ή</reg></choice></hi></abbr><ex>τη</ex><abbr rend="supraline"><hi rend="ligature">ρ</hi></abbr></expan>` which should now be correct. We can test the index against this now.

**Comment by TheocharisPetrou** [comment]
Gabby, with the above updated markup the whole ligature ΜΗΡ of the abbreviation Μή(τη)ρ was separated in two upper ligature lines instead of one as in other examples we' ve done,  e.g. 043.031

**Comment by gabrielbodard** [comment]
@emylonas Can you see if you can fix in (a) the XSLT and CSS *and* (b) the index, the issue of ligatures broken across words, such as two examples in line 5 of [044.053](http://127.0.0.1:9999/en/inscriptions/MedCY.044.053.html) (`<hi rend="ligature" xml:id="lig1">ν</hi> <hi rend="ligature" xml:id="lig2">μ</hi><join target="#lig1 #lig2"/>` and `hi rend="ligature" xml:id="lig3">ὰ</hi>`), which appear in diplomatic as separate loops above N, M, A and Π. They are also indexed as four separate, single-character ligatures.

This is likely to be hard, so if not fixable, (1) we might talk to Irene; (2) we might have to tolerate this bug (in the latter case we can still fix the index display for examples of `<ex>`  inside `<hi>`, as above, which is a rarer instance of this problem).
